evap/evaluation/management/commands/update_course_state.py

```diff
@@ -0,0 +1,14 @@
+import logging
+
+from django.core.management.base import BaseCommand
+
+from evap.evaluation.models import Course
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = 'Updates courses state, when evaluation time begins/ends.'
```

Comment:
"Updates the state of all courses whose evaluation period starts or ends today."